src/models/Settings.js

```diff
@@ -44,6 +44,8 @@ export default Model.extend({
       value: 'auto',
     },
 
+    flow: ['string', false, 'default'],
```

Comment:
add a comment..

// Widget can use this to bootstrap itself with custom login flows instead of standard idx responses

Reply:
added

Reply:
I don't think this is quite correct. The flows are pre-defined (not custom) and not all of them are "login" flows (although most ultimately turn into authenticate flows after some initial steps). 

in interaction_code.md:

```
By default, Widget will either proceed with an existing flow or start a new authenticate flow. This option allows bootstrapping the Widget into a specific view such as register or forgot password. Using this option, it is possible to build specific pages for login, register, and forgot password.

login.html
new OktaSignIn({
  flow: 'login'
})


signup.html
new OktaSignIn({
  flow: 'signup'
})

forgot_password.html
new OktaSignIn({
  flow: 'forgotPassword'
})


```

in Settings.js

// allows bootstrapping the Widget into a specific view such as register or forgot password.

Reply:
login flows - login experience.. but to be specific it's better to say `view`